packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
     dest = temp[temp.length - 1];
   }
 
-  return dest;
+  if (isUUID.anyNonNil(dest)) {
```

Comment:
replace with `/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(dest)`

Reply:
you can make the regex a variable that describes what it does or just leave a comment describing what it's testing for

Reply:
store the regex in the constants file, why are these uuid formats not in the isUUID lib?

Reply:
Its not there in the UUId lib i couldn't find any . Taymore let me check it . There is some 5 version of uuid i guess

Reply:
https://github.com/afram/is-uuid/blob/master/lib/is-uuid.js#L35